src/store/index.js

```diff
@@ -340,8 +370,33 @@ export default new Vuex.Store({
     setPreferredDeliveryMethod(state, payload) {
       state.deliveryMethod = payload;
     },
-    setGlobalAllowConflicts(state, payload) {
-      state.globalAllowConflicts = payload;
+    setGlobalAllowConflicts(state, payload)
```

Comment:
Don't quite get this changes. Can you explain this?

Reply:
Seems to just be a minor formatting change, not a major change. I can change it back if you want